exam -01/section-4.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
```

Comment:
Section 4: This is the work done by Open AI. It's completely ok to use AI to assist your work during the exam but asking AI to do the whole thing while learning is really not good!

Also, this section-4 completely missed the criteria required in the exam:
- Use Class
- Use import / export